obsreport/obsreport.go

```diff
@@ -36,6 +36,11 @@ var (
 	okStatus = trace.Status{Code: trace.StatusCodeOK}
 )
 
+// ObsMetrics wraps OpenCensus View for Collector observability metrics
+type ObsMetrics struct {
+	Views []*view.View
+}
```

Comment:
Can we put this in `internal` so external users cannot construct it?

Reply:
`obsreport.ProcessorMetricViews` is used in a few components in `contrib`. We'll need it for contrib changes.

Reply:
I think we should remove these calls, we should look into why is this called, and how we can get rid of it. I think they are mainly call into the `ProcessorMetricViews` to standardize the name, but we have the `BuildProcessorCustomMetricName` so mostly they can change to just build the name of the metrics with that call.

What do you think?

Reply:
Yes, you're right. I'll make the changes in separate PR since I've made this PR enough now to hide `Configure()` and `AllViews()` from external. Updated description as well.  PTAL! Thanks